javatests/arcs/core/storage/DirectStoreMuxerTest.kt

```diff
@@ -10,24 +22,54 @@ import arcs.core.data.SchemaFields
 import arcs.core.data.util.toReferencable
 import arcs.core.storage.api.DriverAndKeyConfigurator
 import arcs.core.storage.keys.RamDiskStorageKey
+import arcs.core.type.Type
 import com.google.common.truth.Truth.assertThat
+import kotlin.reflect.KClass
 import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.Job
 import kotlinx.coroutines.coroutineScope
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.After
+import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.junit.runners.JUnit4
 
 @RunWith(JUnit4::class)
+@ExperimentalCoroutinesApi
 class DirectStoreMuxerTest {
 
+    private lateinit var storageKey: RamDiskStorageKey
+    private lateinit var schema: Schema
+
+    @Before
+    fun setup() = runBlockingTest {
+        storageKey = RamDiskStorageKey("test")
+        schema = Schema(
+            emptySet(),
+            SchemaFields(
+                singletons = mapOf(
+                    "name" to FieldType.Text,
+                    "age" to FieldType.Int
+                ),
+                collections = emptyMap()
+            ),
+            "abc"
+        )
+    }
+
+    @After
+    fun teardown() {
+        DriverFactory.clearRegistrations()
+    }
+
     @Test
     fun directStoreMuxerNoRace() = runBlocking<Unit>(Dispatchers.IO) {
         DriverAndKeyConfigurator.configure(null)
 
-        val storageKey = RamDiskStorageKey("test")
-
         val schema = Schema(
```

Comment:
This is a different schema. Using the global schema would require making other changes to this test (CrdtSingleton, CrdtEntity, etc.). I decided not to because this is already quite a large PR